### PR TITLE
reorganized settings

### PR DESCRIPTION
**Pull request by Catarina98**

### CODE REVIEW

File: GrocifyApp.UI/src/App.jsx [hunk 1]
```diff
@@ -13,6 +14,8 @@ function App() {
         <div className="container-page">
             <Router>
                 <Routes>
+                    <Route path={AppRoutes.Logout} element={<Logout />} />
+                    <Route index element={isAuthenticated ? <Navigate to={AppRoutes.Settings} /> : <LoginForm />} />
```
Comment (jessywork) [inline GrocifyApp.UI/src/App.jsx]: why did you move settings to here?

File: GrocifyApp.UI/src/components/Logout.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        localStorage.removeItem('token');
+
+        // Redirect to the base page after logout
+        navigate('/');
+    }, [navigate]);
+
+  return (
+    <div>
+      Logging out...
+      {/* You can add a loading spinner or any other UI elements here */}
```
Comment (jessywork) [inline GrocifyApp.UI/src/components/Logout.jsx]: add TODO for implementation later

File: GrocifyApp.UI/src/pages/Settings.jsx [hunk 3]
```diff
@@ -1,134 +1,118 @@
 import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import { ReactSVG } from 'react-svg';
-//import { useNavigate } from 'react-router-dom';
-import { PlaceholderConsts } from '../consts/ENConsts';
+
+//Internal components
```
Comment (jessywork) [inline GrocifyApp.UI/src/pages/Settings.jsx:5]: wow, nice organization... 10 of 10

File: GrocifyApp.UI/src/pages/Settings.jsx [hunk 4]
```diff
@@ -1,134 +1,118 @@
...
+        {
+            tableName: SettingsConsts.Account,
+            items: [
+                { title: SettingsConsts.Logout, icon: LogoutIcon, link: AppRoutes.Logout },
+                { title: SettingsConsts.ClearData, icon: TrashIcon, color: 'color-r300' },
+            ]
+        }
+    ];
+
+    const [searchInput, setSearchInput] = useState(''); //change to darkMode not false
```
Comment (jessywork) [inline GrocifyApp.UI/src/pages/Settings.jsx]: wrong comment

File: GrocifyApp.UI/src/pages/Settings.jsx [hunk 5]
```diff
@@ -1,134 +1,118 @@
...
+                { title: SettingsConsts.MealPlan },
+                { title: SettingsConsts.Meals },
+            ]
+        },
+        {
+            tableName: SettingsConsts.Appearance,
+            items: [
+                {
+                    title: SettingsConsts.DarkMode,
+                    link: () => setDarkMode(), // Pass the function
```
Comment (jessywork) [inline GrocifyApp.UI/src/pages/Settings.jsx]: setDarkMode(!darkMode) ?

**Review by jessywork — CHANGES_REQUESTED** [review]

File: GrocifyApp.UI/src/App.jsx (hunk 1 above)
Comment (Catarina98) [reply GrocifyApp.UI/src/App.jsx]: to be more organized, first route Logout, second: if authenticated: Settings, else: Login

File: GrocifyApp.UI/src/pages/Settings.jsx (hunk 3 above)
Comment (Catarina98) [reply GrocifyApp.UI/src/pages/Settings.jsx:5]: I want to thank chatGPT for all the support, and also my friends and family for never giving up on me.

**Review by jessywork — APPROVED** [review]

**Pull request merged**